src/extension/autoComplete.ts

```diff
@@ -153,11 +153,13 @@ export default class AutoComplete {
           wellKnownContract.manifest;
       }
     } catch (e) {
-      Log.error(
-        LOG_PREFIX,
-        "Error initializing well-known manifests...",
-        e.message
-      );
+      if (e instanceof Error) {
```

Comment:
I assume this is because of https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-4.html#defaulting-to-the-unknown-type-in-catch-variables---useunknownincatchvariables right?

Maybe we should introduce a helper in Log to accept an unknown? Also, while I'm pretty sure VSCode is good about always throwing `Error` objects, maybe we should log `String(e)` if e is not an error?  

Should we introduce a

Reply:
Yes, it was to fix those compile-time errors. I like the log helper idea. I will do that

Reply:
Added check for Error instance in the logger so caller can pass in e directly as suggested